src/App.tsx

```diff
@@ -1,61 +1,138 @@
 import './App.scss';
+import { FormEvent, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { CompletedTodo, HandleEvent, TypeTodo } from './type';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const getUser = (todo: TypeTodo) => {
+  return usersFromServer.find(user => user.id === todo.userId);
+};
+
+const todos: CompletedTodo[] = todosFromServer.map(todo => ({
+  todo,
+  user: getUser(todo),
+}));
 
 export const App = () => {
+  const [stateTodos, setTodos] = useState(todos);
+  const [choosenUser, setChooseUser] = useState('empty');
+  const [newTitle, setNewTitle] = useState('');
+  const [isErrorTitle, setIsErrorTitle] = useState(false);
+  const [isErrorUser, setIsErrorUser] = useState(false);
+  let currentId = stateTodos
+    .reduce((max, { todo }): number => (max < todo.id ? todo.id : max), 0);
+
+  const createTodo = (title: string) => {
+    currentId += 1;
```

Comment:
Still, id add in the wrong way, always 16. You should calculate it inside of `createTodo` - here.
```suggestion
   let currentId = stateTodos
    .reduce((max, { todo }): number => (max < todo.id ? todo.id : max), 0);
```